Tests/Weaver/AssemblyToProcess/TestObjects.cs

```diff
@@ -92,6 +95,9 @@ public class AllTypesObject : RealmObject
         public RealmInteger<long>? NullableInt64CounterProperty { get; set; }
 
         public ObjectId? NullableObjectIdProperty { get; set; }
+
+        public Guid? NullableGuidProperty { get; set; }
+
```

Comment:
```suggestion
```

Reply:
I suppose you wanted to suggest to remove the line after `NullableGuidProperty` and not before. I will do it in a commit :)